packages/utils/telemetry-utils/src/batchedTelemetryHelper.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import type {
+	ITelemetryGenericEventExt,
+	ITelemetryLoggerExt,
+	ITelemetryPerformanceEventExt,
+} from "./telemetryTypes.js";
+
+/**
+ * Expected type of the custom data passed into the logger.
+ */
+type ICustomDataMap = Record<string, number>;
+
+/**
+ * Helper class that accumulates custom data and sends it to the logger every time a specified number (i.e., `threshold`) of logs is reached.
+ */
+export class BatchedTelemetryHelper {
+	disposed: boolean = false;
```

Comment:
Needs a comment on what disposed does. I didn't see where it's used in this class.

Reply:
This should be deleted as I have removed the `dispose()` method. Good catch.